ibc/chain.go

```diff
@@ -16,8 +16,9 @@ type Chain interface {
 	// sets up everything needed (validators, gentx, fullnodes, peering, additional accounts) for chain to start from genesis
 	Start(testName string, ctx context.Context, additionalGenesisWallets ...WalletAmount) error
 
-	// start a chain with a provided genesis file. Will override validators for first 2/3 of voting power
-	StartWithGenesisFile(testName string, ctx context.Context, home string, pool *dockertest.Pool, networkID string, genesisFilePath string) error
+	// Exec runs an arbitrary command using Chain's docker environment.
+	// "env" are environment variables in the format "MY_ENV_VAR=value"
+	Exec(ctx context.Context, cmd []string, env []string) (stdout, stderr []byte, err error)
```

Comment:
FWIW, I think we need to break this interface up into separate categories of chains.

Such as:
```
BaseChain // probably includes IBCTransfer
InterchainAccountChain
QueryChain // IBC queries
SmartContractChain
ExportableChain // export and start from state
```